elixir/apps/domain/lib/domain/auth/provider/query.ex

```diff
@@ -48,7 +48,7 @@ defmodule Domain.Auth.Provider.Query do
       [providers: providers],
       is_nil(providers.last_synced_at) or
         fragment(
-          "? + LEAST((interval '10 minute' * (COALESCE(?, 0) ^ 2 + 1)), interval '4 hours') < NOW()",
+          "? + LEAST((interval '10 minute' * (COALESCE(?, 0) ^ 2 + 1)), interval '4 hours') < timezone('UTC', NOW())",
```

Comment:
This was essentially preventing from running on my local cluster.

Reply:
That's interesting, I've not experienced that on my local dev instance.  Just for my own edification, what was causing the issue?

Reply:
Postgres doesn't take timezone into account when comparing datetimes (!!).

Reply:
So `last_synced_at` was always greater than `NOW()`.